src/Stratis.Bitcoin.Features.Consensus/CoinViews/DBreezeCoinView.cs

```diff
@@ -40,6 +37,9 @@ public class DBreezeCoinView : ICoinView, IDisposable
 
         private BackendPerformanceSnapshot latestPerformanceSnapShot;
 
+        /// <summary>Access to DBreeze database.</summary>
+        public readonly DBreezeEngine DBreeze;
```

Comment:
Instead of exposing this as a public property, I'd create in this class a CreateTransaction method that return a DBreeze Transaction to be used outside. In the future we'd want to use a ITransactionFactory service to inject into constructor of classes that needs a DBreeze Transaction. (copy&pasted observation done where we uses this exposed member)

Reply:
(RewindDataIndexRepository.cs)

Reply:
Fixed